src/components/views/Page.tsx

```diff
@@ -59,16 +68,43 @@ const PageView = ({ children, title, breadcrumbs, actions }: PageViewProps) => {
           </Text>
           {actions && (
             <div className="flex gap-3">
-              {actions.map((action, index) => (
-                <Button
-                  key={`action-${index}`}
-                  onClick={action.onClick}
-                  loading={action.loading}
-                  size="sm"
-                >
-                  {action.content}
-                </Button>
-              ))}
+              <div className="md:hidden relative">
+                <DropdownMenu>
+                  <DropdownMenuTrigger asChild>
+                    <button className="inline-flex items-center justify-center whitespace-nowrap rounded-md focus-visible:outline-none border h-8 w-8 p-0">
+                      <DotsVerticalIcon/>
+                    </button>
+                  </DropdownMenuTrigger>
+                  <DropdownMenuContent
+                    side="bottom"
+                    align="end"
+                    className="absolute right-0 z-50 min-w-[200px] space-y-3"
+                  >
+                    {actions.map((action, index) => (
+                      <DropdownMenuItem
+                        key={`action-${index}`}
+                        onClick={action.onClick}
+                        className="gap-2"
+                      >
+                        <UploadIcon/>
+                        {action.content}
```

Comment:
I'm a little confused as to what you're trying to ask here. Are you suggesting that we remove the loading element and instead make the code the following?
```
<Button
     key={`action-${index}`}
     onClick={action.onClick}
     size="sm"
>
     {action.content}
</Button>
```

Reply:
Wow I wrote the original comment so quickly, skipped so many words.

Anyway, no, what I'm saying is that the actions folded into the dropdown menu could have nontrivial wait times for the thing they do to complete. An example is if you use the desktop version of the students page and import students. It takes a while before completing (on real courses it takes longer). So on mobile with this PR, we kinda lose the observability of knowing that the task is ongoing since the dropdown item component does not have a `loading` prop like the Button component does - i.e. the user might think something went wrong and try to hit the button again because they don't know it's done anything.

Like I mentioned, might not be worth the headache, but worth considering